io/include/pcl/io/image_ir.h

```diff
@@ -55,8 +55,8 @@ namespace pcl
     class PCL_EXPORTS IRImage
     {
       public:
-        using Ptr = shared_ptr<IRImage>;
-        using ConstPtr = shared_ptr<const IRImage>;
+        using Ptr = std::shared_ptr<IRImage>;
```

Comment:
You're all over the places with your shared_ptr changes. Let's undo them all and focus on non-shared-ptr changes.

They have potential to cause ABI breakage and thus it's better to split them to a second PR anyways

Reply:
OK, but a couple of questions:

- Is there a continued need for `pcl::shared_ptr`?
- Shouldn't source code in the `pcl` namespace be enclosed in `namespace pcl { ... }`?
- Removing `std::` from uses of `shared_ptr` will cause some files to not compile. I assume it will be safe to change those instances to `std::shared_ptr`?

Reply:
1. Yes, for sometime more still. Not all of ecosystem has moved.
2. Yes. If it's not, then it's not in `namespace pcl` (and one of my comments might be wrong)
3. Yes. Those are `std::shared_ptr`